pod/perlipc.pod

```diff
@@ -204,12 +204,15 @@ info to show that it works; it should be replaced with the real code.
   my $script = File::Basename::basename($0);
   my $SELF  = catfile($FindBin::Bin, $script);
 
-  # POSIX unmasks the sigprocmask properly
   $SIG{HUP} = sub {
       print "got SIGHUP\n";
-      exec($SELF, @ARGV)        || die "$0: couldn't restart: $!";
+      exec($SELF, @ARGV) || die "$0: couldn't restart: $!";
   };
 
+  # Make sure SIGHUP isn't blocked (as it normally would be, due to the
+  # handler function calling exec rather than returning)
+  POSIX::sigprocmask(&POSIX::SIG_UNBLOCK, POSIX::SigSet->new(&POSIX::SIGHUP));
```

Comment:
This patch is wrong, the unblocking needs to happen inside the signal handler right before the exec.